packages/gateway-client/copy.sh

```diff
@@ -3,6 +3,6 @@
 cd /usr/src
 mv ./volumes/gateway_client/assets/libp2p.* .
 rm -rf ./volumes/gateway_client/*
-cp -rf ./athena/dist/packages/gateway-client/. ./volumes/gateway_client
-mv libp2p.* ./volumes/gateway_client/assets/
+cp -rf ./athena/packages/gateway-client/. ./volumes/gateway_client
+mv libp2p.* /usr/src/volumes/gateway_client
```

Comment:
If you needed to change this line to an absolute path for some reason, we should probably update the other paths to `volumes` in order to remain consistent.